generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user-management.component.ts

```diff
@@ -132,7 +144,20 @@ export class UserMgmtComponent implements OnInit {
         this.users = data;
     }
 
-    private onError (error) {
+    private onError(error) {
         this.alertService.error(error.error, error.message, null);
     }
+
+    createUser() {
+        this.modalRef = this.userModalService.open(UserMgmtDialogComponent);
```

Comment:
may be we can use a route to call a component and open the model from onInit or similar event of the component? we can have that as a thin component on the same file as the model component?